web-console/4.0-designs/taints-tolerations/taints-tolerations.md

```diff
@@ -0,0 +1,27 @@
+# Taints and Tolerations
+
+## Taints
+![taints](img/node-details.png)
+
+![taints2](img/node-taints.png)
```

Comment:
@bmignano I think that the unselected, selected & hover states of the remove icon button should match what we are doing on the Environments screen, where they are gray/black by default, and turn blue on hover.